python/src/puml/gen_puml.aac

```diff
@@ -0,0 +1,131 @@
+            given:
+              - The {{puml.input.architecture-file}} contains a valid model.
+            when:
+              - The aac app is run with the puml-sequence command.
+            then:
+              - A PlantUML sequence diagram of the use case is written to the output directory.
+          - name: Output PlantUML sequence diagram to file.
+            given:
+              - The {{puml.input.architecture-file}} contains a valid model.
+              - The positional argument {{puml.input.output-directory}} contains a valid filepath.
+            when:
+              - The aac app is run with the puml-sequence command.
+            then:
+              - A PlantUML sequence diagram of the model is generated to a .puml file.
+    - name: puml-object
+      help_text: Convert an AaC-defined system to PlantUML object diagram.
+      input:
+        - name: architecture-file
+          type: file
+          description: |
+            A path to a YAML file containing an AaC-defined system from which to
+            generate a PlantUML object diagram.
+        - name: output-directory
+          type: directory
+          description: |
+            The output directory into which the PlantUML (.puml) diagram file
+            will be written.
+      acceptance:
+        name: Object Diagram Test
+        scenarios:
+          - name: Output PlantUML object diagram for valid model.
+            given:
+              - The {{puml.input.architecture-file}} contains a valid model.
+            when:
+              - The aac app is run with the puml-object command.
+            then:
+              - A PlantUML object diagram of the model is written to the output directory.
+          - name: Output PlantUML object diagram to file.
+            given:
+              - The {{puml.input.architecture-file}} contains a valid model.
+              - The positional argument {{puml.input.output-directory}} contains a valid filepath.
+            when:
+              - The aac app is run with the puml-object command.
+            then:
+              - A PlantUML object diagram of the model is generated to a .puml file.
+    - name: puml-requirements
+      help_text: Convert an AaC-defined system to a requirements diagram in PlantUML format.
+      input:
+        - name: architecture-file
+          type: file
+          description: |
+            A path to a YAML file containing an AaC-defined system from which to
+            generate a PlantUML requirements diagram.
+        - name: output-directory
+          type: directory
+          description: |
+            The output directory into which the PlantUML (.puml) diagram file
+            will be written.
+      acceptance:
+        name: Requirement Diagram Test
+        scenarios:
+        - name: Output a requirements diagram in PlantUML format with a valid system model.
+          given:
+            - The {{puml-requirement.input.architecture-file}} contains a valid model.
```

Comment:
I understand that right now the `puml-requirements` is fully separated hence the difference for `puml.input` as in the previous three command acceptance tests and `puml-requirements.input` for the `puml-requirements` command. But the more I have thought on this I don't think that is how it should remain. Either the `puml-requirements` implementation needs included in the main `gen_plant_uml_impl.py`, or each of the diagram options need split out into their own like `puml-requirements` was. The organization of the logic needs to be consistent. 

Thoughts? @bunchw please provide your thoughts as well on this.

Reply:
Yes, we should keep it consistent.  Including the puml-requirements implementation in the gen_plant_uml_impl.py sounds more straightforward to me.